configure-training-env.sh

```diff
@@ -9,6 +9,14 @@ set -eu
 BASE_PATH=~
 KAYOBE_ENVIRONMENT=training
 
+# Check if kayobe password file exists from which to source the password, if it does exist, source it
```

Comment:
I normally wouldn't worry about this sort of thing but given that this is a training project we need to be as clear as possible, particularly because many of the people using this won't speak english as their first language
```suggestion
# Check if a password file exists. If it does, export the contents as the Kayobe vault password environment variable.
```